index.html

```diff
@@ -171,5 +175,76 @@ <h2>Multi-Post Stories</h2>
         </div>
       </div>
     </section>
+    <section class="contact-form">
+      <div class="description">
+        <h1>Contact Us</h1>
+        <p>
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding. I’d love to help
+          with it
+        </p>
+      </div>
+      <form action="https://formspree.io/f/mzbodazz">
```

Comment:
- [ ] Nice job on integrating the formspace service however you forgot to add the `method="POST"` attribute in your form that's why I am getting this 👇 👇  notification when submitting the form.
![image](https://user-images.githubusercontent.com/65068771/150118860-0ce6f1a9-f515-43f3-8f7d-1018fc92bc66.png)

this 👇  is  how it show look like
```html
 <form action="https://formspree.io/f/mzbodazz" method="POST">
    ...content
 <form>

```